hibernate-core/src/main/java/org/hibernate/engine/spi/ActionQueue.java

```diff
@@ -1129,9 +1127,6 @@ private boolean hasParent(BatchIdentifier batchIdentifier, List<BatchIdentifier>
 			}
 		}
 
-		// the mapping of entity names to their latest batch numbers.
-		private List<BatchIdentifier> latestBatches;
```

Comment:
the above should be temporary variable within method, as opposed to a private field